web/themes/custom/atrium/includes/page.inc

```diff
@@ -185,10 +186,30 @@ function _atrium_translation_menu() {
       ? str_replace("/$langcode", '', $current_path)
       : $current_path;
   }
-  // Create links for dropbutton component if translated content exists for page.
+  // Create links for dropbutton component if translated content or webform exists for page.
   $links = [];
   foreach ($languages as $langcode => $language) {
     $node = \Drupal::routeMatch()->getParameter('node');
+    $webform = \Drupal::routeMatch()->getParameter('webform');
```

Comment:
This 2 lines can be taken outside of the foreach loop.

Reply:
Moved lines out of the foreach